sdk/dapp-kit/src/hooks/wallet/useAccounts.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useWalletStore } from './useWalletStore.js';
+
+/**
+ * Retrieves a list of connected accounts authorized by the dApp.
+ */
+export function useAccounts() {
```

Comment:
Added this back since I thought I could get away with removing it in a previous PR